hnginternship5/kidstories-android
# Login reg

### TITLE OF PR

FIXED LOGIN AND REG
### PURPOSE OF PR

To Fix the login and registration to work with new API
### FEATURES IMPLEMENTED

User Login
User registration 
### HOW TO TEST

When a user clicks the Login Button,  the user should be able to log in. 
Also, the user should able to Register.
